src/routes/docs/products/functions/develop/+page.markdoc

```diff
@@ -69,18 +66,14 @@ use Appwrite\Exception;
 return function ($context) {
     // Why not try the Appwrite SDK?
     //
-    // $client = new Client();
-    // $client
-    //     ->setEndpoint('https://cloud.appwrite.io/v1')
-    //     ->setProject(getenv('APPWRITE_FUNCTION_PROJECT_ID'))
-    //      ->setKey(getenv('APPWRITE_API_KEY'));
-
+    // $client = (new Client());
+    // $client->setKey($context.req.headers['x-appwrite-key']); Set API key; it's automatically provided by Appwrite
```

Comment:
Let's keep the multiline like before so we can make it clear the comment is for the `setKey()` part.